csfieldguide/interactives/content/en/interactives.yaml

```diff
@@ -146,3 +152,5 @@ unicode-length:
   name: Unicode Length
 viola-jones-face-detector:
   name: viola-jones-face-detector (broken)
+ziv-lempel-coding:
```

Comment:
The ziv-lempel interactive: regarding earlier comment

Reply:
ooooh here, I was in the /structure/ directory, no wonder I couldn't see it. Yea this is an old reference, will delete.